packages/jest-jasmine2/src/index.js

```diff
@@ -263,12 +263,15 @@ function jasmine2(config, environment, moduleLoader, testPath) {
   env.addReporter(reporter);
   moduleLoader.requireModule(testPath);
   env.execute();
-  env.snapshotState.snapshot.save();
+  const operationResult = env.snapshotState.snapshot.save();
 
   return reporter.getResults().then(results => {
     results.snapshotsAdded = env.snapshotState.added;
     results.snapshotsRemoved = env.snapshotState.removed;
     results.snapshotsMatched = env.snapshotState.matched;
+    if (operationResult.deleted) {
```

Comment:
There may be more than one file removed, also two different scenarios when this may happen, this cover the case when no more `toMatchSnapshot` reference exists in a test file but the same test file still exists.
In this case it can only be deleted or not so either 0 or 1, I should probably initialize it above with 0 for enabling the creation of the hidden class.